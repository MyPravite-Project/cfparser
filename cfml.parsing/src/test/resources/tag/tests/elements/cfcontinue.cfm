<cfloop query="qResults">
    <cfcontinue></cfcontinue>
    <CFCONTINUE>
    <cfcontinue/>
</cfloop>