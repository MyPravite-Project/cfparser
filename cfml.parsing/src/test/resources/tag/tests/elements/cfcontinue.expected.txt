START:cfloop
  =========TAG=================
  <cfloop query="qResults">
    <cfcontinue></cfcontinue>
    <CFCONTINUE>
    <cfcontinue/>
</cfloop>
  START:cfcontinue
    =========TAG=================
    <cfcontinue>
  END:cfcontinue
  START:cfcontinue
    =========TAG=================
    <CFCONTINUE>
  END:cfcontinue
  START:cfcontinue
    =========TAG=================
    <cfcontinue/>
  END:cfcontinue
END:cfloop
